macOS/DuckDuckGo/NavigationBar/View/NavigationBar.storyboard

```diff
@@ -742,6 +713,43 @@
                                     <action selector="clearButtonAction:" target="mcr-nj-GmC" id="m9h-NY-Ujx"/>
                                 </connections>
                             </button>
+                            <stackView distribution="fill" orientation="horizontal" alignment="top" horizontalStackHuggingPriority="249.99998474121094" verticalStackHuggingPriority="249.99998474121094" detachesHiddenViews="YES" translatesAutoresizingMaskIntoConstraints="NO" id="L9C-04-I02">
```

Comment:
Xib magic 😅

* Added the duck.ai button inside a stack view for future proofing and to easily handle the case where it’s not visible.
* Changed the constraints for the bookmark and clear buttons to include horizontal spacing in the new stack view.

Reply:
That sounds good :D thanks for taking time to explain the changes :)